code/game/gamemodes/antagspawner.dm

```diff
@@ -0,0 +1,63 @@
+/obj/item/weapon/antag_spawner
+	throw_speed = 1
+	throw_range = 5
+	w_class = 1.0
+	var/used = 0
+
+/obj/item/weapon/antag_spawner/proc/spawn_antag(var/client/C, var/turf/T, var/type = "")
+	return
+
+/obj/item/weapon/antag_spawner/proc/equip_antag(mob/target as mob)
+	return
+
+/obj/item/weapon/antag_spawner/borg_tele
+	name = "Syndicate Cyborg Teleporter"
+	desc = "A single-use teleporter used to deploy a Syndicate Cyborg on the field."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "locator"
+	var/searching = 0
+
+
+/obj/item/weapon/antag_spawner/borg_tele/attack_self(mob/user)
+	user << "<span class='notice'>The syndicate robot teleporter is attempting to locate an available cyborg.</span>"
+	searching = 1
+	for(var/mob/dead/observer/O in player_list)
+		if(!O.MayRespawn())
+			continue
+		if(jobban_isbanned(O, "Syndicate") && jobban_isbanned(O, "Mercenary"))

```

Comment:
I think you should also check if someone is jobbaned from cyborg, its a cyborg after all